time stamp 
float32 top_temp
float32 bot_temp
